### PR TITLE
Add myself as code owner for debug info generation

### PR DESCRIPTION
**Pull request by adrian-prantl**

### CODE REVIEW

**Comment by adrian-prantl** [comment]
@swift-ci test

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -132,6 +136,7 @@
 # test
 /test/ASTGen/                                       @zoecarver @CodaFi
 /test/Constraints/                                  @hborla @xedin
+swift/test/DebugInfo/                               @adrian-prantl
```
Comment (AnthonyLatsis) [inline .github/CODEOWNERS:139]: Did you mean `/test/DebugInfo/`?

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -98,13 +99,16 @@
...
+/lib/IRGen/*Debug*                                  @adrian-prantl
 /lib/Parse/                                         @ahoppen @bnbarham @CodaFi @DougGregor @rintaro
 /lib/PrintAsClang                                   @zoecarver @hyp @egorzhdan
 # TODO: /lib/SIL/
 # TODO: /lib/SILGen/
 /lib/SILGen/*Distributed*                           @ktoso
 # TODO: /lib/SILOptimizer/
 /lib/SILOptimizer/Utils/Distributed*                @ktoso
+/lib/SIL/IR/SILLocation*                            @adrian-prantl
+/lib/SIL/IR/SILDebug*                               @adrian-prantl
```
Comment (AnthonyLatsis) [inline .github/CODEOWNERS:111]: These should be ordered before `/lib/SILGen` and swapped.

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -98,13 +99,16 @@
 /lib/Index/                                         @bnbarham
 /lib/Refactoring/                                   @ahoppen @bnbarham
 /lib/IRGen/*Distributed*                            @ktoso
+/lib/IRGen/*Debug*                                  @adrian-prantl
 /lib/Parse/                                         @ahoppen @bnbarham @CodaFi @DougGregor @rintaro
 /lib/PrintAsClang                                   @zoecarver @hyp @egorzhdan
```
Comment (AnthonyLatsis) [inline .github/CODEOWNERS:104]: Could you put these in lexicographic order?

**Comment by adrian-prantl** [comment]
@swift-ci smoke test

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -158,6 +164,7 @@
 
 # tools
 # TODO: /tools
+tools/lldb-moduleimport-test/                       @adrian-prantl
 /tools/SourceKit                                    @ahoppen @bnbarham @rintaro
 /tools/swift-ide-test                               @ahoppen @bnbarham @rintaro
```
Comment (AnthonyLatsis) [inline .github/CODEOWNERS:169]: Sorry to be nitpicky.

* The order is ASCII, so lowercase letters are ordered before uppercase ones.
* Could we be consistent with the leading `/`?

**Comment by adrian-prantl** [comment]
@swift-ci smoke test

**Review by AnthonyLatsis — APPROVED** [review]
Thanks!

**Comment by adrian-prantl** [comment]
@swift-ci smoke test

**Pull request merged**